src/AlbumSearch.js

```diff
@@ -28,6 +28,8 @@ function AlbumSearch(){
   const [searchQuery, setSearchQuery] = useState('');
   const [accessToken, setAccessToken] = useState("");
   const [albums, setAlbums] = useState([]);
+  const isLoginSpotify = accessToken !== null;
```

Comment:
Please use context. See Nav.js for example.

Reply:
it's been updated